privaterelay/templates/profile.html

```diff
@@ -400,6 +413,10 @@
       </div> <!-- end informtaion footer -->
       
     </div><!-- end dashboard-container -->
+
+    {% if relay_addresses|length == 0 %}
+      {% include "includes/banners/default-banners.html" %}
```

Comment:
The reason I stored those default banners into its own partial is so I can call it here, where it switches to a bottom placement when a user has at least one alias.